libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -52,8 +52,10 @@ void AP_RCProtocol::init()
     backend[AP_RCProtocol::CRSF] = new AP_RCProtocol_CRSF(*this);
     backend[AP_RCProtocol::FPORT2] = new AP_RCProtocol_FPort2(*this, true);
 #endif
+#if !defined(IOMCU_FW) && HAL_ENABLE_DSHOT
     backend[AP_RCProtocol::ST24] = new AP_RCProtocol_ST24(*this);
     backend[AP_RCProtocol::FPORT] = new AP_RCProtocol_FPort(*this, true);
+#endif
```

Comment:
the shared buffer patch would enable these to get re-enabled

Reply:
No longer needed